src/main/resources/config/application.properties

```diff
@@ -29,25 +29,17 @@ core_case_data.api.url=${CORE_CASE_DATA_API_URL:http://localhost:4452}
 core_case_data.jurisdictionId=${CORE_CASE_DATA_JURISDICTION_ID:SSCS}
 core_case_data.caseTypeId=${CORE_CASE_DATA_CASE_TYPE_ID:Benefit}
 
-# IDAM
+# IDAM Java Client
 idam.url=${IDAM_API_URL:http://localhost:5000}
 idam.s2s-auth.totp_secret=${IDAM_S2S_AUTH_TOTP_SECRET:AAAAAAAAAAAAAAAC}
 idam.s2s-auth.microservice=${IDAM_S2S_AUTH_MICROSERVICE:sscs}
 idam.s2s-auth.url=${IDAM_S2S_AUTH:http://localhost:4502}
 idam.oauth2.user.email=${IDAM_SSCS_SYSTEMUPDATE_USER:system.update@hmcts.net}
 idam.oauth2.user.password=${IDAM_SSCS_SYSTEMUPDATE_PASSWORD:Pa55word11}
-idam.oauth2.client.id=${IDAM_OAUTH2_CLIENT_ID:sscs}
-idam.oauth2.client.secret=${IDAM_OAUTH2_CLIENT_SECRET:QM5RQQ53LZFOSIXJ}
 idam.oauth2.redirectUrl=${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authenticated}
-idam.oauth2.jwkUrl=${IDAM_API_JWK_URL:http://localhost:5000/jwks}
-
 auth.idam.client.baseUrl=${IDAM_API_URL:http://localhost:5000}
-auth.idam.client.id=${IDAM_OAUTH2_CLIENT_ID:sscs}
```

Comment:
I think we have spring security on tribunal api and its expecting naming convention of property start with auth

Reply:
smoke and functional tests seemed to pass on PR. Would that verify?

Reply:
(The new named env variables from previous PR have come from idam-java-client) so assume they must be legit